docs/firststeps/getstarted-new.md

```diff
@@ -25,17 +25,17 @@
 # Get Started
 
 
-To start using the LUMI supercomputer, please follow the steps 1-5:
+To start using the LUMI supercomputer, use must:
```

Comment:
"use must"?
Should it be either "user must", "you must", "one must", or something else

Reply:
Indeed `you must` is what I meant. I have fixed this.